core/templates/domain/utilities/language-util.service.spec.ts

```diff
@@ -181,3 +182,16 @@ describe('Language util service', function() {
     }]);
   });
 });
+
+describe('Language Constants', () => {
+  it('should have no directional descrepencies between Content and Audio' +
```

Comment:
"discrepancies" is misspelled (in two places).

Add a space between "Audio" and "languages".

Reply:
Fixed. Thanks for catching this.